README.md

```diff
@@ -32,7 +32,7 @@ A very minimal config file can look something like this:
 ```js
 module.exports = {
   project_name: 'PROJECT_NAME',
-  plugins: ['loc'],
+  plugins: { loc: true }, // Add metrics about number of lines of code
```

Comment:
I'm trying to follow the same pattern as eslint, except that here all plugins are within the same package, so no need to import, which makes it impossible to just set `plugins: { loc }`. 

https://eslint.org/docs/latest/use/configure/plugins

Anyway, since we expect plugins to provide an options object, I'd rather encourage the pattern with an empty object (instead of true):

```suggestion
  plugins: { loc: {} }, // Add metrics about number of lines of code
```